models/fates/R/model2netcdf.FATES.R

```diff
@@ -10,581 +10,137 @@
 ##' @name model2netcdf.FATES
 ##' @title Code to convert FATES netcdf output into into CF standard
 ##'
-##' @param outdir Location of FATES model output
+##' @param outdir Location of FATES model output (e.g. a path to a single ensemble output)
+##' @param sitelat Latitude of the site
+##' @param sitelon Longitude of the site
+##' @param start_date Start time of the simulation
+##' @param end_date End time of the simulation
+##' @param pfts a named vector of PFT numbers where the names are PFT names
+##'
 ##' 
 ##' @examples  
```

Comment:
please also delete or revise this `examples` bit, the function has more args now than just outdir and this is not how it should be called `model2netcdf.FATES(outdir="~/")`